packages/rke2-snapshot-validation-webhook/package.yaml

```
@@ -1,4 +1,4 @@
 url: https://github.com/piraeusdatastore/helm-charts.git
 subdirectory: charts/snapshot-validation-webhook
-commit: bc135abc9dff8e96cdb9efea56328032391ec0d0  # snapshot-validation-webhook-1.7.1
-packageVersion: 01
+commit: 994dde7e93cda78eaf098e5c39889c95be928b32  # snapshot-validation-webhook-1.7.3
+packageVersion: 02
```

packageVersion should go back to 0 when bumping the release
```suggestion
packageVersion: 00
```